File: dataset/human/java/file_1132.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) throws Exception {
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		long[] a = new long[n];
		a[0] = sc.nextInt();
		for (int i = 1; i < n; i++) {
			a[i] = a[i - 1] + sc.nextInt();
		}
		sc.close();

		long ans = Long.MAX_VALUE;
		for (int i = 0; i < n - 1; i++) {
			long x = a[i];
			long y = a[n - 1] - a[i];
			ans = Math.min(ans, Math.abs(x - y));
		}
		System.out.println(ans);
	}
}

File: dataset/human/cpp/file_1132.cpp
#include <iostream>

using namespace std;

int main()
{
	int r, g, b, n;
	cin >> r >> g >> b >> n;

	int ans = 0;
	for (int i = 0; i <= n; i+=r) {
		for (int j = 0; i + j <= n; j+=g) {
			if ((n - i - j) % b == 0) {
				ans++;
			}
		}
	}
	cout << ans << endl;

	return 0;
}
